src/Impostor.Api/Events/Game/IGameHostChangeEvent.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Impostor.Api.Net;
+
+namespace Impostor.Api.Events
+{
+    public interface IGameHostChangeEvent : IGameEvent
```

Comment:
Event is called after host is changed, so `IGameHostChangedEvent` not `IGameHostChangeEvent`